libvast/vast/view.hpp

```diff
@@ -445,6 +445,14 @@ data_view make_data_view(const T& x) {
   return make_view(x);
 }
 
+/// @relates view_trait
+template <class T>
+data_view make_data_view(const optional<T>& x) {
+  if (!x)
+    return make_view(caf::none);
+  return make_view(x);
```

Comment:
Or simpler: `x ? make_data_view(x) : make_data_view(caf::none)`

Reply:
But, what I want to ask you: don't you want `make_data_view(*x)`? Note the deref!